code/modules/mob/living/silicon/robot/examine.dm

```diff
@@ -3,6 +3,11 @@
 	if(desc)
 		. += "[desc]"
 
+	// SKYRAT EDIT START - Borg models in their examine text
+	var/model_name = model ? "\improper [model.name]" : "\improper Default"
+	. += "\nIt is currently \a \"[model_name]\"-type cyborg.\n"
```

Comment:
Might be easier to just put 
```suggestion
	. += "\nIt is currently \a \[model_name]\ model.\n"
```
Appearing as "It is currently a Department model." More concise and still gets the point across

Alternatively we could clean some models out, we've got quite a few of varying standards and they should at the least follow departmental colors

Reply:
I decided to go with this, because I couldn't find a nice way to word it that contained "model" without it looking too weird. That's why I decided to go with the way I've done it.

I'm not saying that I'm against some re-coloring of a few of them to follow departmental colors better, but I think this is going to be useful regardless.